tests/lib/rules/indent.js

```diff
@@ -21,57 +21,390 @@ const ruleTester = new RuleTester({
 });
 
 ruleTester.run("indent", rule, {
-    valid: [
-        `
-@Component({
-    components: {
-        ErrorPage: () => import('@/components/ErrorPage.vue'),
-    },
-    head: {
-        titleTemplate(title) {
-            if (title) {
-                return \`test\`
-            }
-            return 'Title'
+    valid: [],
+    invalid: [
+        /*
```

Comment:
i'm not sure if this is intended but most of tests is commented

Reply:
ugh. yeah i'm a spaz.. i did that to reduce the noise whilst testing and forgot about it.